scheduler/server/job_state.go

```diff
@@ -4,7 +4,9 @@ import (
 	"math"
 	"time"
 
+	lru "github.com/hashicorp/golang-lru"
 	log "github.com/sirupsen/logrus"
+
```

Comment:
nit: blank line

Reply:
added for consistency with other files (separate twitter/scoot pkgs from 3rdparty pkgs)